docs/src/developer/standards.rst

```diff
@@ -1,47 +1,94 @@
-structure of the source repository, and finally readme_renderer_ which
-checks whether ``/README.rst`` can be
-actually rendered in PyPI_ website page for Oríon.
+In addition, we follow `Numpy's docstring standards
+<https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>`_ to ensure a good
+quality of documentation for the project.
 
-.. code-block:: sh
+.. _standard-vcs:
 
-   tox -e packaging
+Version Control Guidelines
+==========================
 
-To run all of expected linters execute::
+To collaborate through VCS, we follow the
+`gitflow <https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow>`_
+workflow. The *develop* and *master* branches are protected and can only be changed with pull
+requests.
 
-   tox -e lint
+For branch names, we recommend prefixing the name of the branch with *feature/*, *fix/*, and
+*doc/* depending on the change you're implementing. Additionally, we encourage adding the issue's id
+(if there is one) at the start of the branch name, after the prefix. For example the branch name for
+a bug represented by issue 225 would be ``fix/225-short-bug-description``.
 
-.. _flake8: http://flake8.pycqa.org/en/latest/
-.. _pylint: https://www.pylint.org/
-.. _check-manifest: https://pypi.org/project/check-manifest/
-.. _readme_renderer: https://pypi.org/project/readme_renderer/
-.. _PyPI: https://pypi.org/
+When creating a release, we use the pattern *release-{version}rc*. This branch represent the release
+candidate that will be merged in the master branch when the changes are ready to be launched in
+production.
+
+Synchronization
+---------------
+Regarding merges, we recommend you keep your changes in your forked repository for as long as
+possible and rebase your branch to Oríon's develop branch before submitting a pull request.
+
+Most probably, the develop branch will have changed by the time your pull request is approved. In
+such cases, we recommend that you merge the changes from develop to your branch when the reviewer
+approves your pull request and then the maintainer will merge your branch to develop, closing your
+pull request.
+
+We discourage rebases after the pull resquest has been submitted as it can cause problems in
+GitHub's review system which makes it loose track of the comments on the pull request. On another
+note, merges are always done with the creation of a merge commit, also known as a *non fast-forward
+merge*.
+
+In some cases where the pull request embodies contributions which are scattered across multiple
+commits containing incremental changes (e.g., ``fix pep8``, ``update based on feedback``), the pull
+request may be integrated to the development branch using `squash and merge <https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits>`_
+to avoid clutter. It is strongly encouraged to make small pull requests. They are simpler to
```

Comment:
I would add `by the maintainer [to avoid clutter]`. Otherwise it may be confusing for the contributor, they may think this sentence means they could do the squash and merge.